include/onnxruntime/core/common/logging/logging.h

```diff
@@ -219,6 +219,8 @@ class LoggingManager final {
   const bool default_filter_user_data_;
   const int default_max_vlog_level_;
   bool owns_default_logger_;
+
+ public:  // TODO: Made public only for the provider bridge. Should be changed to a method to get this instead
```

Comment:
Do you intend to fix this?

Reply:
Looks like it wasn't even needed now! Reverted